src/anki/ankiclient.cpp

```diff
@@ -1770,24 +1765,33 @@ QString AnkiClient::buildFrequencies(const QList<Frequency> &frequencies)
     return freqStr;
 }
 
-QString &AnkiClient::accumulateTags(const QList<Tag> &tags, QString &tagStr)
+void AnkiClient::buildTags(const QList<Tag> &tags, QString &tagStr, QString &tagBriefStr)
 {
     if (tags.isEmpty())
-        return tagStr;
+    {
+        tagStr.clear();
+        tagBriefStr.clear();
+        return;
+    }
 
+    tagStr      = "<ul>\n";
```

Comment:
`\n` is unnecessary in markup and isn't used anywhere else.

Reply:
I forgot to explain this, but the newline was on purpose. When using CSS to transform the list to be horizontal, having <li> be separated by newlines adds space around them, while they would be merged together otherwise. But I suppose one could use padding or ::after to add space instead, which would be a more reliable solution.

Reply:
It's best not to rely on the content of the markup when configuring the presentation. Thanks for changing it.